.github/workflows/ci.yml

```
@@ -0,0 +1,48 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
```

Add nightly CRON with schedule and on demand runs with workflow dispatch

```suggestion
  # Run daily at 0:01 UTC
  schedule:
  - cron:  '1 0 * * *'
  workflow_dispatch:
```

---

I am ok with `workflow_dispatch`, as it is something convenient.

Now, when does a cron-job CI run going to detect issues (in format, or test compliance), that neither the PR creation triggers, nor the `master` push trigger will detect?

---

You'd want to know ahead of time if changes in dependency releases are going to affect you.

---

That is a good point!

I think that is useful when there is the possibility that **installation** requirements could break. Right now, [PR #1](https://github.com/HEPData/hepdata-pyhf-extractor/pull/1) does not introduce any _installation_ requirements, only _development_ ones.

We could then setup this cronjob rule for the future, although I am not sure what installation requirements may come down the way. Perhaps `pyyaml`? 🤷🏻‍♂️